app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
I suggest taking advantage of this page a lot more. You can put  the common sitewide header and navigation links here and they'll show up on each page.  **Very** handy.